target/linux/mediatek/dts/mt7981b-routerich-ax3000.dts

```diff
@@ -0,0 +1,337 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
mt7981* files are licensed using GPL 2.0 **and** MIT. To make this clear and fully compatible please relicense your file to both licenses. Add `or MIT` to the `SPDX` header.

Totally clear would be:
```
// SPDX-License-Identifier: GPL-2.0-only OR MIT
```

---

I guess you could also try
```
// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```
but that doesn't make much sense give included file isn't licensed as "or later".

Reply:
@rmilecki done.